src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.html

```diff
@@ -90,12 +90,11 @@ <h4 jhiTranslate="artemisApp.modelingSubmission.assessment.title">Assessment</h4
                                         <span *ngIf="feedback.reference">{{ assessmentsNames[feedback.referenceId!]['type'] }} </span>
                                         <code *ngIf="feedback.reference" class="text-primary">{{ assessmentsNames[feedback.referenceId!]['name'] }}</code
                                         ><br *ngIf="feedback.reference" />
-                                        <span *ngIf="feedback.text" [ngClass]="{ 'text-success': feedback.credits! > 0, 'text-danger': feedback.credits! < 0 }"
-                                            ><strong>Feedback:</strong> {{ feedback.text }}</span
-                                        >
-                                        <span *ngIf="feedback.detailText" [ngClass]="{ 'text-success': feedback.credits! > 0, 'text-danger': feedback.credits! < 0 }"
-                                            ><strong>Feedback:</strong> {{ feedback.detailText }}</span
-                                        >
+                                        <span
+                                            *ngIf="feedback.text || feedback.detailText || feedback.gradingInstruction"
+                                            [ngClass]="{ 'text-success': feedback.credits! > 0, 'text-danger': feedback.credits! < 0 }"
+                                            ><strong>Feedback:</strong><span [innerHTML]="buildFeedbackTextForReview(feedback)"></span
```

Comment:
I think there could be a space added after `Feedback:`
 the result for the modeling exercise assessment is:
<img width="249" alt="Bildschirmfoto 2021-11-06 um 23 37 33" src="https://user-images.githubusercontent.com/41366522/140625799-26bc57a4-07aa-4add-9ade-14f295112200.png">

Reply:
Done!